Documentation/reportqueries.md

```diff
@@ -34,18 +34,18 @@ Most of these functions are for referring to other resources such as `ReportData
 
 #### Query Inputs
 
-Within the template context, the `.Report.Inputs` template variable contains a map where each key is the name of an input defined by the ReportGenerationQueries `spec.inputs`.
+Within the template context, the `.Report.Inputs` template variable contains a map where each key is the name of an input defined by the ReportQueries `spec.inputs`.
 The value of an input depends on a if the user specified a value in their `Report` or `ReportDataSource`, or if there is a default value defined.
 
-- Depending on what is referencing the ReportGenerationQuery: The value comes from a `Report`'s `spec.inputs` or from a `ReportDataSource`'s `spec.generationQueryView.inputs`.
-- If not provided, it will use the `default` value as defined the the ReportGenerationQueries `spec.inputs`, if one exists.
+- Depending on what is referencing the ReportQuery: The value comes from a `Report`'s `spec.inputs` or from a `ReportDataSource`'s `spec.reportQueryView.inputs`.
+- If not provided, it will use the `default` value as defined the the ReportQueries `spec.inputs`, if one exists.
```

Comment:
Same thing here, but also there's a duplicate "the", `as defined the the ReportQueries`